sympy/physics/control/tests/test_lti.py

```diff
@@ -505,6 +507,28 @@ def test_Series_construction():
     raises(TypeError, lambda: Series(tf3, Matrix([1, 2, 3, 4])))
 
 
+def test_PIDController_instance():
+    KP, KI, KD = 1, 0.1, 0.01
+    pid = PIDController(KP, KI, KD, s)
+    assert isinstance(pid, PIDController), "The object should be an instance of PIDController."
```

Comment:
The error message is not required. You can remove all of them.

Reply:
okay sir